addon/ng2/tasks/install.js

```diff
@@ -0,0 +1,178 @@
+/* jshint node: true, esnext: true */
```

Comment:
As a general rule, tasks should perform a single task and be composed of other tasks. Those other tasks are reusable.

Ask yourself how you can define the `install` task into steps, and those steps should become their own tasks. Stuff like installing typings should be its own task.